test/unit/modules/accounts.js

```diff
@@ -0,0 +1,725 @@
+'use strict';/*eslint*/
+
+var node = require('./../../node.js');
+var ed = require('../../../helpers/ed');
+var bignum = require('../../../helpers/bignum.js');
+var crypto = require('crypto');
+var async = require('async');
+var sinon = require('sinon');
+
+var chai = require('chai');
+var expect = require('chai').expect;
+var constants = require('../../../helpers/constants.js');
+var _  = require('lodash');
+
+var Rounds = require('../../../modules/rounds.js');
+var AccountLogic = require('../../../logic/account.js');
+var AccountModule = require('../../../modules/accounts.js');
+var TransactionLogic = require('../../../logic/transaction.js');
+var TransactionModule = require('../../../modules/transactions.js');
+var DelegateModule = require('../../../modules/delegates.js');
+var BlocksModule = require('../../../modules/blocks.js');
+var modulesLoader = require('../../common/initModule').modulesLoader;
+
+
+var accountSecret = 'actress route auction pudding shiver crater forum liquid blouse imitate seven front';
+var validAccount = {
```

Comment:
I think it's okay for now. I'll work on a new PR which'll consolidate all the test data we require in one place.